packages/quick-edit/README.md

```diff
@@ -0,0 +1,36 @@
+# VSCode for Web
+
+This package contains Cloudflare's fork VSCode for Web, to support web editing of Workers. Before working with this package, you'll need to run `./setup.sh` in `packages/quick-edit`, which will clone VSCode (currently v1.7.6), apply the patches specified in `./patches`, and symlink the top level packages within `workers-sdk`. This package primarily contains code and setup for _building_ VSCode for web, but it can also be used for development.
+
+## Developing
+
+1. You must switch your node version to Node 16 (using a tool like nvm). VSCode's build process requires this.
```

Comment:
it may be good to include a `.node-version` file to make this requirement a little more explicit (and consumable by some version managers)

Reply:
also consider adding an [`engines`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines) property to the `package.json` file to emit a warning on `npm install` if you're using the wrong version of node